__tests__/app.test.js

```diff
@@ -165,6 +165,93 @@ describe("GET /api/articles/:article_id", () => {
 
 
 
+describe("GET /api/articles", () => {
+  test("Get:200 return  all articles ", () => {
+    return request(app).get("/api/articles")
+      .expect(200);
+  });
+
+  test("responds with article data in array", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toBeInstanceOf(Array);
+      });
+  });
+
+  test(" return a length of article array", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.length).toBe(13);
+      });
+  });
+
+ test("should return all the articles with correct properties", () => {
```

Comment:
These probably could have been grouped together in one test. That being said, this is really thorough and you're checking exactly what should be tested✅